docs/cudf/source/api_docs/series.rst

```diff
@@ -59,9 +60,6 @@ Indexing, iteration
    Series.iteritems
    Series.keys
 
-For more information on ``.at``, ``.iat``, ``.loc``, and
-``.iloc``,  see the :ref:`indexing documentation <indexing>`.
```

Comment:
Are indexing docs gone?

Reply:
They never existed, as far as I can tell!

I searched through the history from `branch-0.14` to `branch-22.04` and never found the file. I suspect this was [copied from pandas](https://github.com/pandas-dev/pandas/blame/76eb314a0ef53c907da14f74a3745c5084a1428a/doc/source/reference/frame.rst#L80-L81) and never checked. Originally added in PR #8746.

Reply:
I see, my mistake. Sorry for that, should we also be having a similar page to https://pandas.pydata.org/docs/user_guide/indexing.html#indexing? Not in the scope of this PR but we can track in an open issue.

Reply:
I don't think we need it. If we do need it, we could have an outbound link to the pandas docs and avoid maintaining such a page.